CFC_WebApp/api/cfc_webapp.py

```diff
@@ -1,4 +1,3 @@
-import json
```

Comment:
@lechiang, remove outcome of patch.
Easiest way to do this is 
git checkout api/cfc_webapp.py instead of manually reverting changes.